small changes (#1246)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

Registering new provider within an extension
============================================

> There has to be a provider, no matter whether variables or functions shall be provided.

Reviewer: I'm not sure but I kind of remember that "shall" should (heh) be avoided ...

Author: ==> will.

       page.10.value = Matched
   [GLOBAL]

> In the case when a simple string like a title is returned, this can be further checked:

Reviewer: ```suggestion
If a simple string like a title is returned, this can be further checked:
```
Neither version is correct apparently:

https://dictionary.cambridge.org/grammar/british-grammar/in-case-of

Author: undone

       page.10.value = Matched
   [GLOBAL]

> In case a simple string like a title is returned, this can be further checked:

Reviewer: ```suggestion
If a simple string like a title is returned, this can be further checked:
```

Author: If a simple string like a page title is returned, this can be further compared:
